src/java/com/rapleaf/jack/queries/WhereConstraint.java

```diff
@@ -54,6 +54,6 @@ public Logic getLogic() {
 
   @Override
   public String getSqlStatement() {
-    return (logic != null ? logic.toString() + " " : "") + "`" + column.getSqlKeyword() + "` " + operator.getSqlStatement();
```

Comment:
ok, I am adding it back and I am making my change dirtier.
I will create a seperate PRs to support backticks and double quotes